docs/unpackaged.rst

```diff
@@ -1,14 +1,212 @@
+
+.. note:: To learn more about which components are packageable, see the `Metadata Coverage Report <https://mdcoverage.secure.force.com/docs/metadata-coverage>`_.
+
+..
+
+    ``unpackaged/post`` is the home for this kind of metadata, which is stored in subdirectories such as ``unpackaged/post/first``.
+
+CumulusCI's standard flows that build orgs, such as ``dev_org`` and ``install_prod``, always deploy metadata bundles found in ``unpackaged/post``, making it a full-fledged part of the application. It's also easy to include ``unpackaged/post`` metadata in customer-facing installers run via MetaDeploy.
+
+The ``deploy_post`` task, which is part of the ``config_dev``, ``config_qa``, and ``config_managed`` flows, is responsible for deploying these bundles.
+
+.. important:: Do not include metadata in ``unpackaged/post`` unless it is intended to be delivered to *all* environments (both managed installations and unmanaged deployments). It's also critical for managed package projects that this metadata include namespace tokens (see `namespace injection`_).
+
+
+``unpackaged/config``: Tailor an Org
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Projects can come with more than one supported configuration in their CumulusCI automation.
+
+    Example: Projects often support distinct, tailored ``dev_org``, ``qa_org``, and ``install_prod`` flows, each of which performs a unique setup for their specific use case.
+
+Unpackaged metadata stored in ``unpackaged/config`` is a tool to support operational needs that tailor orgs to different configurations. 
+
+    Example: A testing-oriented scratch org needs to deploy a customized set of Page Layouts to help testers easily visualize data under test. Such page layouts are stored in ``unpackaged/config/qa``.
+
+
+
+Unpackaged Metadata Folder Structure
+------------------------------------
+
+All unpackaged metadata is stored in the ``unpackaged`` directory tree, which contains these top-level directories.
+
+* ``unpackaged/pre``
+* ``unpackaged/post``
+* ``unpackaged/config``
+
+These trees contain metadata bundles in Metadata API format, represented as a directory containing a ``package.xml`` manifest and Metadata API-format source code. CumulusCI does not support Salesforce DX format for unpackaged bundles.
+
+
+
+Namespace Injection
 -------------------
 
-Customize config flows
+Projects that build managed packages often construct their unpackaged metadata to be deployable in multiple contexts.
+
+* Unmanaged deployments, such as developer orgs
+* Unmanaged namespaced scratch orgs
+* Managed contexts, such as a beta test org or a demo org created with ``install_prod``
+
+Because projects that are building an org implementation or a non-namespaced package do not have a namespace, or a distinction between managed and unmanaged contexts, these projects must also build namespaced packages.
```

Comment:
Why must they build namespaced packages? I would like to cap this sentence with just a few words.

"These projects must also build namespaced packages to __________________."

Reply:
@davidmreed - Is a non-namespaced package just another way of saying 'unamangeg/unlocked' package? Also wondering about Justin's question here. The following example outlines how `unpackaged/post` needs to work in both managed and unmanaged contexts; however, if I'm building an org implementation in a package without a namespace, what's the reason for needing to build namespaced packages?

Reply:
I think I wrote this sentence, but it's total nonsense. 

```suggestion
Because projects that are building an org implementation or a non-namespaced package do not have a namespace, or a distinction between managed and unmanaged contexts, these projects typically don't need to use namespace injection.
```